ext/skeleton/.gitignore.in

```diff
@@ -15,7 +15,6 @@ config.status
 config.sub
 configure
 configure.ac
-configure.in
```

Comment:
I think this one should stay for now as usually exts will be compatible with more than only PHP 7.2. Depending on the used version a different file will be used.

Reply:
I see... Reverted. Thanks for this info.